content/source/docs/enterprise-beta/sentinel/enforce.html.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: enterprise2
+page_title: "Sentinel - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-sentinel-enforce"
+---
+
+# Enforce and Override Policies
+
+Once a policy is added to an organization it is enforced on all Workspace Runs.
+
+The policy check will occur immediately after a plan is successfully executed in the run. If the plan fails, the policy check will not be performed. The policy check uses the generated tfplan file along with the state and configuration to verify the rules in each of the policies.
```

Comment:
I would note the expected apply object too.

Reply:
What do you mean by "expected apply object"?

Reply:
Sorry that language might not be the best. Basically the "simulated apply object"

Reply:
https://docs.hashicorp.com/sentinel/app/terraform/import-tfplan#resource-applied-field

Reply:
A link to the `tfplan` plugin docs would be great here.